src/Diagnostics.RuntimeHost/Services/StorageService/StorageService.cs

```diff
@@ -57,11 +58,15 @@ public StorageService(IConfiguration configuration, IHostingEnvironment hostingE
             {
                 var accountname = configuration["SourceWatcher:DiagStorageAccount"];
                 var key = configuration["SourceWatcher:DiagStorageKey"];
-                var storageAccount = new CloudStorageAccount(new StorageCredentials(accountname, key), accountname, "core.windows.net", true);
+                var storageAccount = new CloudStorageAccount(new StorageCredentials(accountname, key), accountname, "core.windows.net", true);        
```

Comment:
will this work for any other cloud as we have the DNS suffix hardcoded here?

Reply:
Valid point, It wont work. I will change it